recipes-graphics/xorg-lib/libxkbcommon_0.8.0.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Generic XKB keymap library"
+DESCRIPTION = "libxkbcommon is a keymap compiler and support library which \
+processes a reduced subset of keymaps as defined by the XKB specification."
+HOMEPAGE = "http://www.xkbcommon.org"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e525ed9809e1f8a07cf4bce8b09e8b87"
+LICENSE = "MIT & MIT-style"
+
```

Comment:
Do you know why the compiler finds the original parser.h first along with the newer parser.c?
Also, have newer releases of the project updated to use a newer bison?

Reply:
> Do you know why the compiler finds the original parser.h first along with the newer parser.c?
I haven't looked closely at this but it is probably the order of include paths thats the issue.

> Also, have newer releases of the project updated to use a newer bison?
Newer releases don't have `parser.c` and `parser.h` anymore and yes they have been building fine on hardknott where we have newer bison